diagnostic.md

```diff
@@ -21,13 +21,17 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+curl GET http://www.example.com/donuts;
```

Comment:
curl --include --request GET http://www.example.com/donuts

The request won't work without the --include and --request